File: 3209.java
import java.util.Scanner;

public class Main{
    public static void main(String [] args)
    {
        Scanner scanner = new Scanner(System.in);
        int test = scanner.nextInt();
        scanner.nextLine();

        for (int i = 0; i < test; i++) {
            String[] tomadas = scanner.nextLine().split(" ");
            int lines = Integer.parseInt(tomadas[0]), soma = 0;

            for (int j = 1; j < tomadas.length; j++) {
               soma += Integer.parseInt(tomadas[j]);
            }
            soma = soma - (lines - 1);

            System.out.println(soma);
        }
        
        scanner.close();
    }
}

File: javascript/3209.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto){ return lines.shift(); };
const test = parseInt(prompt("casos de test"));

for (let i = 0; i < test; i++) {
   let tomadas = prompt("linhas").split(" ").map(Number);
   let lines = tomadas.shift(), soma = 0;

   for (let i = 0; i < tomadas.length; i++) {
      soma += tomadas[i];
   }
   soma = soma - (lines - 1)

   console.log(soma);
}
